lib/src/components/NxCombobox/NxCombobox.tsx

```diff
@@ -51,7 +51,8 @@ function NxComboboxRender<T extends string | number = string>(
       newAttrs = omit(['trimmedValue'], attrs),
 
       isEmpty = !matches.length,
-      isAlert = loading || loadError || isEmpty,
+      showEmptyMessage = isEmpty && value.length,
```

Comment:
for clarity, why is showEmptyMessage a boolean when empty, and a number when a value is included?

Reply:
> for clarity, why is showEmptyMessage a boolean when empty, and a number when a value is included?

It's only intended to be used as a boolean-ish value, I just didn't think to `!!` the `value.length`

Reply:
> > for clarity, why is showEmptyMessage a boolean when empty, and a number when a value is included?
> 
> It's only intended to be used as a boolean-ish value, I just didn't think to `!!` the `value.length`

 wanted to confirm if this is good to be left as is, or if it should be changed?

Reply:
> wanted to confirm if this is good to be left as is, or if it should be changed?

I'm ok with it, but can change it if you or someone else wants it changed.

Reply:
> > wanted to confirm if this is good to be left as is, or if it should be changed?
> 
> I'm ok with it, but can change it if you or someone else wants it changed.

I'm ok with it, just wanted to confirm.